src/components/TopBar/index.tsx

```diff
@@ -217,7 +218,8 @@ const TopBar = () => {
               color="brand.primary"
               border="none"
               marginTop={3}
-              as={TriangleUpIcon}
+              as={RiArrowUpSFill}
+              size={25}
```

Comment:
Might need a negative margin or anything to place this icon at the right place.
<img width="562" alt="image" src="https://user-images.githubusercontent.com/12379895/194816499-f617f774-35c1-43b7-84d0-001dbe53026a.png">